interpreters/ficl/src/nuttx.h

```diff
@@ -1,3 +1,27 @@
+/***************************************************************************
+ * apps/interpreters/ficl/src/nuttx.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ***************************************************************************/
+
```

Comment:
Probably need to add
```
#ifndef __APPS_INTERPRETERS_FICL_SRC_NUTTX_H
#define __APPS_INTERPRETERS_FICL_SRC_NUTTX_H
```
as well. But may be done as a separate PR in mainline

Reply:
I think that we are going outside the scope of this commit

Reply:
yes. wrote "done as a separate PR in mainline", so just pointing it here for tracking.
I was thinking to add it to mainline by myself, but I think that this PR should be cherry-picked to mainline in prior to other changes.